### DISCUSSION TITLE
Response was recieved at http://example.com over https://example.com

### DISCUSSION THREAD

**Discussion by sikandder1996** [codegreencreative/laravel-samlidp#108] [Q&A]
after successful login , on redirecting back i recieve the
 error "The response was received at http://abc.test.com/saml2/74bb02aa-fc9e-48eb-990e-549f27e0e107/acs instead of https://abc.test.com/saml2/74bb02aa-fc9e-48eb-990e-549f27e0e107/acs"

**Comment by upwebdesign** [comment]
@sikandder1996, looks like you have an http vs https issue.

**Reply by sikandder1996** [reply]
RelayState has https url but still happens to send response to http

**Reply by sikandder1996** [reply]
proxyVars = true resolves it

**Reply by upwebdesign** [reply]
@sikandder1996 where did you set `proxyVars = true`?
